spec.html

```diff
@@ -7298,6 +7287,17 @@ <h1>ECMAScript Function Objects</h1>
             The <emu-xref href="#sec-source-text">source text</emu-xref> that defines the function.
           </td>
         </tr>
+        <tr>
+          <td>
+            [[CallThrowsTypeError]]
```

Comment:
I'm not too crazy about the name here, as it describes the "how" rather than the "what". I think I would prefer the more conservative approach of naming this `[[IsClassConstructor]]`. We can always generalize the name further in the future.

Reply:
This was changed as a result of my feedback [here](https://github.com/tc39/ecma262/pull/1546#discussion_r287008007).

Reply:
Ah, sorry I missed that.

Reply:
I'm still a bit worried that the new name makes the following text awkward:

> If a built-in constructor has any [[Call]] behaviour other than throwing a *TypeError* exception, an ECMAScript implementation of the function must be done in a manner that does not cause the function's [[CallThrowsTypeError]] internal slot to have the value *true*.

Reply:
It reads a bit stumbly, but it does seem to describe things correctly - if a constructor doesn't throw on call, it's "call throws" slot can't be true.

Reply:
Yeah, I thought that sentence made more sense after the change from`[[IsClassConstructor]]` to `[[CallThrowsTypeError]]`.